File: Testoutside/Framework/Info.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity.Spatial

<Table("Info")>
Partial Public Class Info
    Public Property id As Integer

    <Required>
    <StringLength(50)>
    Public Property name As String

    Public Property age As Integer
End Class

File: Testoutside/Models/Info.cs
namespace TestOutside.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Info")]
    public partial class Info
    {
        public int id { get; set; }

        [Required]
        [StringLength(50)]
        public string name { get; set; }

        public int age { get; set; }
    }
}
